docs/usage/updates-strategy.md

```diff
@@ -65,6 +65,53 @@ Such an approach is only recommended where nodes are already grouped into an orc
 [airlock]: https://github.com/coreos/airlock
 [etcd3]: https://etcd.io/
 
+# File-based strategy
+The `marker_file` strategy is a simple, low-level strategy that only allows Zincati to reboot for updates when a specific marker file exists on the local filesystem.
+
+Similar to the `fleet_lock` strategy, the `marker_file` strategy provides a large amount of flexibility to admins, and can be used with a central controller.
+Unlike `fleet_lock`, where the central controller must be a lock-manager on the network, the central controller for the `marker_file` strategy can be a containerized agent, some central task manager able to manipulate files on machines (e.g. Ansible), or even a human via SSH.
+
+To indicate that a machine is allowed to finalize an update and reboot, a file with the following properties must be present on the machine's local filesystem:
+    - named `allowfinalize.json`
+    - under `/var/lib/zincati/admin/strategy/marker_file`
+    - is a valid JSON file
+    - owned by root
+    - not writable by others
+
+If any of the above is not satisfied in your marker file, Zincati will not allow reboots.
+
+`allowfinalize.json` can optionally contain a `allowUntil` key with a Unix timestamp integer as its value to indicate the expiry date and time of this marker file. If the current time timestamp is _greater than or equal to_ this timestamp, then reboots will not be allowed.
+Otherwise, if the `allowUntil` key is not present, reboots will be allowed for as long as `allowfinalize.json` exists (in the right location), and it must be removed to disallow reboots.
```

Comment:
```
Otherwise, if the `allowUntil` key is not present, reboots will be allowed for as long as `allowfinalize.json` exists (in the right location), and it must be removed to disallow reboots.
```

Can we get an example of that? is it just `echo '{}' | sudo tee /var/lib/zincati/admin/strategy/marker_file/allowfinalize.json` ?

Reply:
Yep, `echo '{}' | sudo tee /var/lib/zincati/admin/strategy/marker_file/allowfinalize.json` would work. I'll add an example :)